doc/release-notes/rl-2511.section.md

```diff
@@ -18,6 +18,7 @@
 - `base16-builder` node package has been removed due to lack of upstream maintenance.
 - `gentium` package now provides `Gentium-*.ttf` files, and not `GentiumPlus-*.ttf` files like before. The font identifiers `Gentium Plus*` are available in the `gentium-plus` package, and if you want to use the more recently updated package `gentium` [by sil](https://software.sil.org/gentium/), you should update your configuration files to use the `Gentium` font identifier.
 - `space-orbit` package has been removed due to lack of upstream maintenance. Debian upstream stopped tracking it in 2011.
+- `command-not-found` package is now disabled by default; it works only for nix-channels based systems, and requires setup for it to work.
```

Comment:
This should have go into the NixOS release notes `nixos/doc/manual/release-notes`, not the nixpkgs ones. Can you make a follow up change?

Reply:
Ah thanks for letting me know, I've moved it to the correct place in https://github.com/NixOS/nixpkgs/pull/432023